package.json

```diff
@@ -54,6 +54,14 @@
   },
   "prettier": {},
   "resolutions": {
-    "**/@octokit/core": "4.2.0"
+    "**/@octokit/core": "4.2.0",
+    "glob-parent": "^5.1.2",
+    "node-fetch": "^2.6.7",
+    "minimist": "^1.2.8",
+    "mixme": "^0.5.2",
+    "minimatch": "^3.1.1",
+    "lodash": "^4.17.21",
+    "trim": "^0.0.3",
```

Comment:
This one comes from `remark-parse` and to fix it we'd have to update a major of this library. I agree that it's best to do it in a separate PR.